app/controllers/keywords_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
 
   def create
     if keywords_parse_csv
+      SearchKeywordsJob.perform_later(keywords_form.insert_keywords)
```

Comment:
In rails, we have some callbacks, do you wanna try this out?
Reference: https://guides.rubyonrails.org/active_record_callbacks.html#creating-an-object

Then we can move this line out of the create action.
This will make us easier for testing 🖖